Assets/VRTK/Scripts/VRTK_MoveInPlace.cs

```diff
@@ -333,6 +339,41 @@ private void FixedUpdate()
             rigidBody.MovePosition(rigidBody.transform.position + movement);
         }
 
+        private Quaternion determineAverageControllerRotation() {
```

Comment:
Uppercase Method name should be `DetermineAverageControllerRotation`

bracket on new line like

```
func
{
}
```